poetry2conda/convert.py

```diff
@@ -8,7 +8,7 @@
 from poetry2conda import __version__
 
 
-def convert(file: TextIO, include_dev=False) -> str:
+def convert(file: TextIO, include_dev=False, extras=None) -> str:
```

Comment:
Can you please add the type annotations to this function?
I believe these would be `include_dev: bool = False, extras: Iterable[str]`

Also, please update the docstring accordingly.

Reply:
done.